TestProjects/VisualEffectGraph_HDRP/Packages/manifest.json

```diff
@@ -1,60 +1,60 @@
-    "com.unity.ide.vscode": "1.2.4",
-    "com.unity.inputsystem": "1.3.0",
-    "com.unity.render-pipelines.core": "file:../../../com.unity.render-pipelines.core",
-    "com.unity.render-pipelines.high-definition": "file:../../../com.unity.render-pipelines.high-definition",
-    "com.unity.render-pipelines.high-definition-config": "file:../../../com.unity.render-pipelines.high-definition-config",
-    "com.unity.shaderanalysis": "file:../../../com.unity.shaderanalysis",
-    "com.unity.shadergraph": "file:../../../com.unity.shadergraph",
-    "com.unity.test-framework": "1.1.30",
-    "com.unity.test-framework.build": "0.0.1-preview.14",
-    "com.unity.test-framework.utp-reporter": "1.1.0-preview",
-    "com.unity.testing.graphics-performance": "file:../../../com.unity.testing.graphics-performance",
-    "com.unity.testing.visualeffectgraph": "file:../../../com.unity.testing.visualeffectgraph",
-    "com.unity.testing.xr": "file:../../../com.unity.testing.xr",
-    "com.unity.timeline": "1.7.0-pre.1",
-    "com.unity.ugui": "1.0.0",
-    "com.unity.visualeffectgraph": "file:../../../com.unity.visualeffectgraph",
-    "com.unity.modules.ai": "1.0.0",
-    "com.unity.modules.androidjni": "1.0.0",
-    "com.unity.modules.animation": "1.0.0",
-    "com.unity.modules.assetbundle": "1.0.0",
-    "com.unity.modules.audio": "1.0.0",
-    "com.unity.modules.cloth": "1.0.0",
-    "com.unity.modules.director": "1.0.0",
-    "com.unity.modules.imageconversion": "1.0.0",
-    "com.unity.modules.imgui": "1.0.0",
-    "com.unity.modules.jsonserialize": "1.0.0",
-    "com.unity.modules.particlesystem": "1.0.0",
-    "com.unity.modules.physics": "1.0.0",
-    "com.unity.modules.physics2d": "1.0.0",
-    "com.unity.modules.screencapture": "1.0.0",
-    "com.unity.modules.terrain": "1.0.0",
-    "com.unity.modules.terrainphysics": "1.0.0",
-    "com.unity.modules.tilemap": "1.0.0",
-    "com.unity.modules.ui": "1.0.0",
-    "com.unity.modules.uielements": "1.0.0",
-    "com.unity.modules.umbra": "1.0.0",
-    "com.unity.modules.unityanalytics": "1.0.0",
-    "com.unity.modules.unitywebrequest": "1.0.0",
-    "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
-    "com.unity.modules.unitywebrequestaudio": "1.0.0",
-    "com.unity.modules.unitywebrequesttexture": "1.0.0",
-    "com.unity.modules.unitywebrequestwww": "1.0.0",
-    "com.unity.modules.vehicles": "1.0.0",
-    "com.unity.modules.video": "1.0.0",
-    "com.unity.modules.vr": "1.0.0",
-    "com.unity.modules.wind": "1.0.0",
-    "com.unity.modules.xr": "1.0.0"
-  },
-  "testables": [
-    "com.unity.testing.visualeffectgraph",
-    "com.unity.testing.xr",
-    "com.unity.render-pipelines.visualeffectgraph",
-    "com.unity.testframework.graphics",
-    "com.unity.testing.graphics-performance"
-  ]
-}
+    "disableProjectUpdate": true,
+    "dependencies": {
+        "com.unity.ide.visualstudio": "2.0.7",
+        "com.unity.ide.vscode": "1.1.3",
+        "com.unity.inputsystem": "1.0.2",
```

Comment:
It seems that some packages were downgraded, is that intended?

Reply:
dont worry, the perf configs already have UTF at 1.1.30 :)